tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -1,10 +1,26 @@
+from urllib.parse import urlparse
+
 import pytest
 from .common import *  # NOQA
+from .test_rke_cluster_provisioning import rke_config, engine_install_url, node_template_linode, \
+    validate_rke_dm_host_2_no_cluster_cleanup
+
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+AWS_REGION = os.environ.get("AWS_REGION")
+AWS_SUBNET = os.environ.get("AWS_SUBNET")
+AWS_VPC = os.environ.get("AWS_VPC")
+AWS_SG = os.environ.get("AWS_SG")
+AWS_ZONE = os.environ.get("AWS_ZONE")
+AWS_IAM_PROFILE = os.environ.get("AWS_IAM_PROFILE", "")
+AWS_S3_BUCKET_NAME = os.environ.get("AWS_S3_BUCKET_NAME", "")
+AWS_S3_BUCKET_FOLDER_NAME = os.environ.get("AWS_S3_BUCKET_FOLDER_NAME", "")
+LINODE_ACCESSKEY = os.environ.get('RANCHER_LINODE_ACCESSKEY', "None")
```

Comment:
@izaac can we moved all the AWS and LINODE related env variables to common.py so that they can reside in 1 place. Also remove them from test_rke_cluster_provisioning.py.